src/pages/scale.tsx

```diff
@@ -47,6 +47,7 @@ export function Scale({
   }
 
   let focusedHex: string | undefined;
+  const index = String(Math.min(+selectedIndex, scale.colors.length - 1));
```

Comment:
[Minor] I'd prefer to be a bit more explicit when casting `selectedIndex` to a number:

```suggestion
  const index = String(Math.min(Number(selectedIndex), scale.colors.length - 1));
```

Reply:
I absolutely agree with you 👍 Fixed that as well 😊